capi_image_builder/builder/git_ops.py

```diff
@@ -43,11 +43,14 @@ def git_fetch_upstream(self, remote_name: str = "upstream"):
         """Fetch the upstream remote of a git repo."""
         self.repo.remotes[remote_name].fetch()
 
-    def git_rebase_upstream(
-        self, remote_name: str = "upstream", branch: str = "master"
-    ):
+    def git_merge_upstream(self, remote_name: str = "upstream", branch: str = "master"):
         """
         Rebase the current branch onto the upstream branch to bring fork into sync.
```

Comment:
Just need to update the description for the method to `Merge the current branch` instead of `rebase`